src/main/java/com/educare/unitylend/dao/CommunityRepository.java

```diff
@@ -0,0 +1,35 @@
+package com.educare.unitylend.dao;
+
+import com.educare.unitylend.model.Community;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Select;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Mapper
+@Repository
+
+public interface CommunityRepository {
```

Comment:
Add extra field in the Community table as community_type (i.e - Village, School, etc.)

Reply:
It has been done.